packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue

```diff
@@ -152,8 +156,13 @@ export default {
 
             if (available) {
                 if (this.authToken) {
-                    this.activationState = ACTIVATION_STATE_AVAILABLE_LOGGED_IN;
                     this.extractConsumerDetails();
+
```

Comment:
Lot of nested `if` statements IMHO here

Reply:
@matthewhardern Yeah, agree. Tried to simplify a bit.